app/jobs/update_efile_errors_job.rb

```diff
@@ -0,0 +1,15 @@
+class UpdateEfileErrorsJob < ApplicationJob
+  def perform
+    ctc_errors = EfileError.where('created_at < ?', Date.new(2024))
+    state_file_errors = EfileError.where('created_at > ?', Date.new(2024))
+
+    ctc_errors.update_all(product_type: :ctc)
+    state_file_errors.update_all(expose: false)
```

Comment:
I think changing to expose false should actually happen before we deploy, there are only currently 4 state_file errors could do this manually even in the hub

Reply:
What's the idea of the expose attribute here? Not sure I understand.  Expose on the hub as opposed to keeping it in the DB?

Reply:
@rickreyhsig Expose on the efile error means that we will expose that error to the client. In state-file's case if expose is true then we are exposing the efile error description and resolution (what's next text) to the client on the return-status page.